pom.xml

```diff
@@ -106,7 +106,12 @@
         <metrics.version>4.2.4</metrics.version>
         <javassist.version>3.21.0-GA</javassist.version>
         <shell-executable>bash</shell-executable>
-        <checkstyle.version>3.1.2</checkstyle.version>
+        <checkstyle.plugin.version>3.1.2</checkstyle.plugin.version>
+        <checkstyle.version>8.45</checkstyle.version>
+        <hugegraph-rpc.version>2.0.1</hugegraph-rpc.version>
+        <hugegraph-common.version>2.1.2</hugegraph-common.version>
```

Comment:
todo: we need use a unify version for rpc & common (or just include commons package maybe)